packages/descendants/src/Descendants.spec.tsx

```diff
@@ -312,5 +285,58 @@ describe('packages/descendants', () => {
         expect(descendant.index).toEqual(i);
       }
     });
+
+    test('Accessing descendants from a callback are not stale', async () => {
+      const logDescendants = jest.fn();
+
+      const Parent = ({
+        children,
+        open,
+        ...rest
+      }: ComponentProps<'div'> & { open: boolean }) => {
+        const { descendants, dispatch, getDescendants } =
+          useInitDescendants<HTMLDivElement>();
+
+        const handleTransition = () => {
+          logDescendants(getDescendants());
+        };
+
+        return (
+          <StrictMode>
```

Comment:
Why do you need to add `StrictMode`? I thought tests were already using `StrictMode`